packages/card/src/vaadin-card.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2024 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { css, html, LitElement } from 'lit';
+import { defineCustomElement } from '@vaadin/component-base/src/define.js';
+import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
+import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+/**
+ * <vaadin-card> is a visual content container.
+ *
+ * ```html
+ * <vaadin-card class="flex flex-col overflow-hidden">
```

Comment:
I'd suggest to keep this example minimal and not use any CSS classes here since users who are unfamiliar with Lumo utility classes might think they are somehow part of the component. Not a blocker though.

Reply:
Yeah, sounds fair. I’ll try to think of an example that doesn't need utility classes on the card itself.

Reply:
Changed to minimal example for now, we can update JSDoc in a separate PR later.